tests/integration/proxy/grpcproxy/cluster_test.go

```diff
@@ -67,6 +69,43 @@ func TestClusterProxyMemberList(t *testing.T) {
 	if mresp.Members[0].ClientURLs[0] != cts.caddr {
 		t.Fatalf("mresp.Members[0].ClientURLs[0] expected %q, got %q", cts.caddr, mresp.Members[0].ClientURLs[0])
 	}
+
+	//test proxy member up/down
+	newMemberAddr := "127.0.0.2:6789"
+	delay := time.Second * time.Duration(1)
+	ttl := 1
+	wait := delay + time.Second*time.Duration(ttl+2)
+	memberDownC := registDelayDeleteMember(lg, serverEps, newMemberAddr, delay, ttl, t)
+
+	//check add member succ
+	mresp, err = client.Cluster.MemberList(context.Background())
+	if err != nil {
+		t.Fatalf("err %v, want nil", err)
+	}
+	if len(mresp.Members) != 2 {
+		t.Fatalf("len(mresp.Members) expected 2, got %d (%+v)", len(mresp.Members), mresp.Members)
+	}
+	if mresp.Members[1].ClientURLs[0] != newMemberAddr {
+		t.Fatalf("mresp.Members[1].ClientURLs[0] expected %q, got %q", newMemberAddr, mresp.Members[1].ClientURLs[0])
+	}
+
+	//check del member succ
+	time.Sleep(wait)
```

Comment:
`wait` > `delay`. What is the point if `delay`?

Reply:
use delay then test case can check proxy member has successfully updayed